src/components/link/link.ts

```diff
@@ -0,0 +1,99 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { LINK_TYPES, LINK_TARGETS } from './defs';
+
+import LinkStyles from './link.scss';
+
+/**
+ * Component for navigation links.
+ * @fires on-click - Captures the click event and emits the original event details.
+ * @slot unnamed - Slot for link text/content.
+ */
+
+@customElement('kd-link')
+export class Link extends LitElement {
+  static override styles = LinkStyles;
+
+  /** Link url. */
+  @property({ type: String })
+  href = '';
+
+  /** Defines a target attribute for where to load the URL. Possible options include "_self" (deafult), "_blank", "_parent", "_top" */
+  @property({ type: String }) target = LINK_TARGETS.SELF;
+
+  /** The Link type. Primary(App) or Secondary(Web).*/
+  @property({ type: String }) kind = LINK_TYPES.PRIMARY;
+
+  /** Defines a relationship between a linked resource and the document. An empty string (default) means no particular relationship */
+  @property({ type: String })
+  rel = this.target === '_blank' ? 'noopener' : '';
```

Comment:
If this prop is tied to the target prop, let's `@ignore` it to hide it from the docs for the user.

Reply:
I have made this **rel = ' '**